chapter2/tests/t20/terraform/dev/_alb.tf

```diff
@@ -0,0 +1,89 @@
+module "alb" {
+  source             = "terraform-aws-modules/alb/aws"
+  version            = "~> 8.0"
+  name               = local.labels.alb
+  load_balancer_type = "application"
+  vpc_id             = data.aws_vpc.target.id
+  subnets            = data.aws_subnets.ec2.ids
+  security_groups    = [module.alb_sg.security_group_id]
+  tags               = var.tags
+
+  target_groups = [
+    {
+      backend_protocol = "http"
+      backend_port     = 80
+      target_type      = "instance"
+        health_check = {
+        enabled             = true
+        interval            = 30
+        path                = "/"
+        port                = "traffic-port"
+        healthy_threshold   = 5
+        unhealthy_threshold = 2
+        timeout             = 6
+        protocol            = "HTTP"
+        matcher             = "200-399"
+      }
+      targets = {
+        my_target_one   = {
+          target_id = module.ec2_instances["one"].id
+          port       = 80
+        }
+        my_target_two   = {
+          target_id = module.ec2_instances["two"].id
+          port       = 80
+        }
+      }
+    }
+  ]
+
+  # http_tcp_listeners = [
+  #   {
+  #     port               = 80
+  #     protocol           = "HTTP"
+  #     target_group_index = 0
+  #   }
+  # ]
```

Comment:
if you don't need - don't keep it commented.

Reply:
> if you don't need - don't keep it commented.

@dmitry-mightydevops 
Ok, done